app/src/main/java/org/akvo/flow/dao/SurveyDbAdapter.java

```diff
@@ -171,11 +176,12 @@
     }
 
     public interface SurveyInstanceStatus {
-        int SAVED      = 0;
-        int SUBMITTED  = 1;
-        int EXPORTED   = 2;
-        int SYNCED     = 3;
-        int DOWNLOADED = 4;
+        int UNKNOWN = 0;
```

Comment:
I got scared and I have reverted back since I am not sure 100% that it may not break something that we will not see immediately. I have in project to refactor the database class and then I will test it fully.